test/yamatanooroti/test_run_ruby.rb

```diff
@@ -44,6 +44,16 @@ def test_meta_key
     get_out_from_tmpdir
   end
 
+  def test_assert_screen_takes_a_message_when_failed
+    start_terminal(5, 30, ['ruby', '-e', 'puts "aaa"'])
+    close
+    assert_raise Test::Unit::AssertionFailedError do
```

Comment:
Let's use [`assert_raise_with_message`](https://github.com/test-unit/test-unit/blob/bdd1668e1e3c0875fd1b3a59df8f6ec6472014a1/lib/test/unit/assertions.rb#L318) so we can also verify that the message is passed along?

Reply:
Right. I used Regexp because the message came with a backtrace and I couldn't confirm an exact match.